kaska/tests/test_reproject.py

```diff
@@ -1,25 +1,24 @@
 #!/usr/bin/env python
 """Test reproject data"""
-import os
 
+import os
 import sys
-sys.path.append("../")
-
+import numpy as np
 import pytest
 import gdal
-import numpy as np
+from ..utils import reproject_data
 
+sys.path.append("../")
```

Comment:
Try the tests without this line. Believe the test should run without it.

Reply:
Yes they do